website/src/css/custom.css

```diff
@@ -524,17 +744,237 @@ code {
   flex: 1 0 300px;
   max-width: 300px;
 }
+
 .blog-main-row .blog-aside li a {
   font-size: 1rem;
   line-height: 1.25;
 }
+
+/* footer styles */
+.footer {
+  font-weight: var(--ifm-font-weight-narrow);
+  font-size: 20px;
+  padding-top: var(--ifm-footer-padding-vertical);
+  padding-bottom: var(--ifm-footer-padding-vertical);
+}
+
+.footer a.footer__link-item {
+  color: var(--ifm-footer-color);
+}
+
+.footer a {
+  color: var(--ifm-footer-color);
+  text-decoration: none;
+}
+
+.footer a:hover {
+  text-decoration: underline;
+  color: var(--ifm-footer-color);
+}
+
+.footer__links {
+  margin-bottom: 1.25em;
+}
+
+.footer__link-item {
+  display: flex;
+  justify-content: space-between;
+}
+
+.footer__items {
+  list-style: none;
+  margin-bottom: 0;
+  padding-left: 0;
+  gap: 1.5em;
+  display: flex;
+}
+
+/* social media icons in footer */
+.footer__items--right {
+  display: flex;
+  gap: 1.5em;
+}
+
+/* fontawesome icons in footer */
+.footer .fa-brands {
+  color: var(--color-footer-accent);
+  font-size: 1.5rem;
+  transition: all .3s ease-in-out;
+}
+
+.footer .fa-brands:hover {
+filter: opacity(.75);
+}
+
+/* copyright */
+.footer__bottom {
+  text-align: left;
+  color: var(--color-footer-accent);
+  font-size: 1rem;
+}
+
+/* Begin @Media Queries */
+@media (min-width: 997px) and (max-width: 1150px) {
+  nav.navbar a.navbar__link {
+    font-size: 0.9rem;
+  }
+
+  .navbar__search .navbar__search-input {
+    max-width: 160px;
+  }
+}
+
+@media (max-width: 910px) {
+  .footer__items {
+    flex-direction: column;
+    align-items: flex-start;
+  }
+}
+
+@media (max-width: 996px) {
+
+  .navbar-sidebar__brand .navbarSidebarToggle_node_modules-\@docusaurus-theme-classic-lib-next-theme-Navbar-styles-module {
+    margin-right: 0;
+    margin-left: .25em;
+  }
+
+  .navbar-sidebar__brand > div:first-of-type {
+    order: 1;
+    margin-right: 0;
+  }
+
+  button.DocSearch-Button {
+    background: var(--ifm-color-gray-100);
+  }
+
+  [data-theme='dark'] button.DocSearch-Button {
+    background-color: black;
+  }
+}
+
+@media screen and (max-width: 700px) {
+  .card.large {
+    height: auto;
+    text-align: center;
+  }
```

Comment:
The CTA text on the cards turn dark when in dark mode. It feels like the text should remain white as the rest of the card styles do not change. This also occurs on the blog header card which uses the same styles.

![image](https://user-images.githubusercontent.com/13647110/190257143-3ecdae54-bc2b-4b43-8e2a-a7790b597391.png)
![image](https://user-images.githubusercontent.com/13647110/190257235-b023c875-5b5a-494d-a341-26b461a9af04.png)

Reply:
should be working now:
https://deploy-preview-1990--docs-getdbt-com.netlify.app/